solr/solrj/src/java/org/apache/solr/common/cloud/VMParamsAllAndReadonlyDigestZkACLProvider.java

```diff
@@ -30,6 +30,12 @@
 
   public static final String DEFAULT_DIGEST_READONLY_USERNAME_VM_PARAM_NAME = "zkDigestReadonlyUsername";
   public static final String DEFAULT_DIGEST_READONLY_PASSWORD_VM_PARAM_NAME = "zkDigestReadonlyPassword";
+
+  // fallback env vars if system props not set
+  public static final String ZK_ALL_ACL_USERNAME = "ZK_ALL_ACL_USERNAME";
+  public static final String ZK_ALL_ACL_PASSWORD = "ZK_ALL_ACL_PASSWORD";
+  public static final String ZK_READ_ACL_USERNAME = "ZK_READ_ACL_USERNAME";
+  public static final String ZK_READ_ACL_PASSWORD = "ZK_READ_ACL_PASSWORD";
```

Comment:
`zkDigestReadonly*` is mentioned in `zookeeper-access-control.adoc` - maybe also mention these alternatives there?

Reply:
will do!